src/components/crossDevice/Intro/index.js

```diff
@@ -11,44 +11,47 @@ import { localised } from '../../../locales'
 const previousComponentType = ({flow = 'captureSteps', documentType, poaDocumentType, steps, step}) =>
   componentsList({ flow, documentType, poaDocumentType, steps })[step || 0].step.type
 
+const getStageIcon = (key, isFace) => {
+  const iconPrefix = 'stageIcon'
+  if (key !== 'take-photos') {
+    return `${iconPrefix}-${key}`
+  }
+  return isFace ? `${iconPrefix}-take-selfie` : `${iconPrefix}-take-photos`
+}
 
 const Intro = ({translate, nextStep, mobileConfig}) => {
   const isFace = previousComponentType(mobileConfig) === 'face'
-  const stages = {
-    'sms': 'sms',
-    ...(isFace ?
-      {'take-selfie': 'face.take_photos' } :
-      {'take-photos': 'document.take_photos' }
-    ),
-    'return-computer': 'return_computer',
+  const stageList = [ 'sms', 'take-photos', 'return-to-computer' ]
```

Comment:
[Minor] If you define `stageListCopyByKey` first, this can only be `const stageList = Object.keys(stageListCopyByKey)`.

Reply:
Was worried about the order not being guaranteed, but with modern JS it should be. 🤔